utils/cloudfront.go

```diff
@@ -153,7 +153,7 @@ func (d *Distribution) setDistributionConfigOrigins(config *cloudfront.Distribut
 	}
 }
 
-func (d *Distribution) setDistributionConfigDefaultCacheBehaviour(config *cloudfront.DistributionConfig, callerReference string) {
+func (d *Distribution) setDistributionConfigDefaultCacheBehavior(config *cloudfront.DistributionConfig, callerReference string) {
```

Comment:
Can we squash this commit in to the previous one?

Reply:
Yes, I want to squash everything into one or two commits once it has been reviewed

The big diff on its own is hard for me to look at, so I wanted to make sure reviewers had it easier with smaller commits showing small changes.